okapi-core/src/main/raml/okapi.raml

```diff
@@ -1128,6 +1128,31 @@ types:
         body:
           text/plain:
 
+/_/proxy/import/modules:
+  description: Import modules
+  post:
+    description: Import list modules.
+    body:
```

Comment:
Please copy the queryParameters documentation from /_/proxy/modules.

Reply:
ok